docs/reference_guide.md

```diff
@@ -1108,6 +1108,14 @@ Attaching 1 probe...
 WARNING: could not determine instruction boundary for uprobe:bin:4377 (binary appears stripped). Misaligned probes can lead to tracee crashes!
 ```
 
+When tracing libraries, it is sufficient to specify the library name instead of
+a full path.
```

Comment:
Maybe also mention the lookup is done via ldcache? This may be important for systems that have multiple versions of a library installed.

Reply:
dont forget to mirror this to the manpage too

Reply:
Good points. Updated, please check.